test/support/utils.rb

```diff
@@ -29,15 +31,13 @@ def debug_code(program, boot_options: '-r debug/run', remote: true, &block)
         debug_on_unix_domain_socket
         debug_on_tcpip
       end
-
-      check_line_num!(program)
     end
 
-    DEBUG_MODE = false
+    ENV['RUBY_DEBUG_TEST_DEBUG_MODE'] ||= 'false'
     ASK_CMD = %w[quit delete kill undisplay]
 
     def debug_print msg
-      print msg if DEBUG_MODE
+      print msg unless ENV['RUBY_DEBUG_TEST_DEBUG_MODE'] == 'false'
```

Comment:
thanks 👍

Reply:
`if ENV['RUBY_DEBUG_TEST_DEBUG_MODE']` ?

Reply:
I changed it.

Reply:
https://github.com/ruby/debug/pull/129/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R39